tests/PHPStan/Analyser/data/getenv-php74.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace GetenvPHP74;
+
+use function PHPStan\Testing\assertType;
+
+class Foo
+{
+
+	public function test()
+	{
+		assertType('string|false', getenv(null));
+		assertType('array<string, string>', getenv());
+		assertType('string|false', getenv('foo'));
+	}
```

Comment:
Please add a test for an argument that's both, something like `getenv($mixed)`. To both PHP 7.4 and PHP 8.0.